tests/unit/lcos/barrier.cpp

```diff
@@ -12,12 +12,15 @@
 
 #include <atomic>
 #include <cstddef>
+#include <cstdint>
 #include <functional>
 #include <string>
+#include <utility>
 #include <vector>
 
 ///////////////////////////////////////////////////////////////////////////////
-void barrier_test(std::size_t num, std::size_t rank, std::atomic<std::size_t>& c)
+void barrier_test(
```

Comment:
Are changes to barrier related to the PR? I suspect a bit of reformatting (whitespace) has crept in. Since you're adding something new, it might be better to remove unrelated changes from the patch.

Reply:
Sorry, by mistake. I removed.